src/classes/Commands.ts

```diff
@@ -321,6 +325,63 @@ export = class Commands {
         );
     }
 
+    private messageCommand(steamID: SteamID, message: string): void {
+        const isAdmin = this.bot.isAdmin(steamID);
+        const parts = message.split(' ');
+        const details = this.bot.friends.getFriend(steamID);
+
+        if (!this.bot.getAlertTypes().includes('message')) {
+            this.bot.sendMessage(steamID, 'Messages are disabled.');
```

Comment:
Instead of doing it like this, then maybe hide the message command / disable it for the user if the owner has disabled it.

Reply:
More like this right? 
`    messageCommand(steamID: SteamID, message: string): void {
        if (process.env.ENABLE_MESSAGE === 'false') {
            this.bot.sendMessage(
                steamID,
                'Manual review is disabled, enable it by setting `ENABLE_MESSAGE` to true'
            );
            return;
        }
        
        const isAdmin = this.bot.isAdmin(steamID);
        const parts = message.split(' ');
        const details = this.bot.friends.getFriend(steamID);`
.
.
.

Reply:
No, this would send that message to both admins and normal users.